LazyProxy/LazyProxyBuilder.cs

```diff
@@ -220,7 +220,8 @@ private static TypeBuilder AddMethods(this TypeBuilder typeBuilder, Type type, F
                     generator.Emit(OpCodes.Ldarg, i);
                 }
 
-                generator.Emit(OpCodes.Call, method);
+                generator.Emit(OpCodes.Tailcall);
```

Comment:
Why is it needed here?

Reply:
sequence   `tailcall, call[virt], ret`   translates into more efficient code than `call[virt], ret`,  probably into **JMP** x86 instruction  instead of **CALL**

Last call before **ret** always can be made **tailcall**

Reply:
Will it influence on the call stack? Could you please check the call stack is correct in this case?

Reply:
Checked it.
The stacks are identical for both cases (with & without **tailcall**). They don't include **LazyProxy** frame. Its look like JIT optimizes it into tail-call independently of **tailcall** prefix.

Reply:
Great. Thank you.